packages/fuels/src/cli/commands/deploy/createWallet.test.ts

```diff
@@ -30,4 +30,16 @@ describe('createWallet', () => {
     expect(error).toBeTruthy();
     expect(error?.message).toMatch(/you must provide.+privateKey.+PRIVATE_KEY/i);
   });
+
+  test('throws error when failed to connect to node', async () => {
+    const providerUrl = 'http://localhost:1234';
```

Comment:
```suggestion
    const providerUrl = 'nope';
```
Is there any risk of #1356 using this port and this test being flaky. Would be pretty fluky, just a thought.

Reply:
Changed it to `http:localhost:0` in https://github.com/FuelLabs/fuels-ts/pull/1854/commits/b786b9bec7d0eab6009e51f5e25db3b766600d53 because the test is failing if it isn't a valid address. Nothing can be hosted on the 0 port so it achieves the same purpose.